src/read.rs

```diff
@@ -48,10 +48,10 @@ mod tests {
     #[test]
     fn test() {
         let events = vec![
-            start_tag!("h1", attrs!["id" => "hello"]),
+            start_tag!("h1", attrs![ id: "hello", class: "fun" ]),
```

Comment:
Sounds good, I was really unsure on how fancy to make these macros; anything really nice like horrorshow I think should be done as a separate crate, but some minor syntax improvements for these would be good.